src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx

```diff
@@ -62,12 +62,12 @@ const ChangeOperatorName = () => {
     return (
         <BorderScreen
             blackHeader
-            header={'Change Operator Name'}
+            header={ 'Change Operator Name' }
```

Comment:
If you take a look at all the { & } places in the code - there is no suffix and postfix spaces, lets keep the same approach everywhere

Reply:
done